usaspending_api/download/delta_models/account_download.py

```diff
@@ -0,0 +1,391 @@
+        transaction_search.parent_uei AS recipient_parent_duns,
+        transaction_search.parent_recipient_name AS recipient_parent_name,
+        transaction_search.parent_recipient_name_raw AS recipient_parent_name_raw,
+        transaction_search.recipient_location_country_code AS recipient_country,
+        transaction_search.recipient_location_state_code AS recipient_state,
+        transaction_search.recipient_location_county_name AS recipient_county,
+        transaction_search.recipient_location_city_name AS recipient_city,
+        transaction_search.pop_country_name AS primary_place_of_performance_country,
+        transaction_search.pop_state_name AS primary_place_of_performance_state,
+        transaction_search.pop_county_name AS primary_place_of_performance_county,
+        transaction_search.place_of_performance_zip4a AS primary_place_of_performance_zip_code,
+        transaction_search.cfda_number,
+        transaction_search.cfda_title,
+        transaction_search.product_or_service_code,
+        transaction_search.product_or_service_description AS product_or_service_code_description,
+        transaction_search.naics_code,
+        transaction_search.naics_description,
+        transaction_search.national_interest_action AS national_interest_action_code,
+        transaction_search.national_interest_desc AS national_interest_action,
+        submission_attributes.reporting_agency_name AS reporting_agency_name,
+        CASE
+            WHEN submission_attributes.quarter_format_flag = TRUE
+                THEN
+                    CONCAT(
+                        CAST('FY' AS STRING),
+                        CAST(submission_attributes.reporting_fiscal_year AS STRING),
+                        CAST('Q' AS STRING),
+                        CAST(
+                            submission_attributes.reporting_fiscal_quarter AS STRING
+                        )
+                    )
+            ELSE
+                CONCAT(
+                    CAST('FY' AS STRING),
+                    CAST(submission_attributes.reporting_fiscal_year AS STRING),
+                    CAST('P' AS STRING),
+                    LPAD(
+                        CAST(
+                            submission_attributes.reporting_fiscal_period AS STRING
+                        ),
+                        2,
+                        '0'
+                    )
+                )
+        END AS submission_period,
+        treasury_appropriation_account.budget_function_title AS budget_function,
+        treasury_appropriation_account.budget_subfunction_title AS budget_subfunction,
+        financial_accounts_by_awards.transaction_obligated_amount AS transaction_obligated_amount,
+        CASE
+            WHEN
+                (
+                    (
+                        submission_attributes.quarter_format_flag = TRUE
+                        AND submission_attributes.reporting_fiscal_quarter = 4
```

Comment:
You are really close to this SQL being done. This download query should not filter by anything that would be input by a user. In this case the Fiscal Quarter, Period, and Year fall into that category. Similar to how the filtering by Fiscal Year had to be moved into the next query, these fields do as well.

Reply:
Similar comment applies to anywhere that is filtering by a specific Fiscal Period or Quarter.